source/layouts/blog_article.erb

```diff
@@ -4,6 +4,15 @@
 <% content_for(:meta_keywords) { "unboxed consulting, news, technical, web, development, marketing, graphic design, Unboxed, Ruby on Rails, Blog, #{current_page.data.tags.join(', ')}" } %>
 <% add_atom_feed_to_head %>
 
+<!--Twitter card tags-->
+<% content_for(:head) do %>
+  <meta name="twitter:card" content="summary_large_image" />
+  <meta name="twitter:site" content="@Ubxd">
+  <meta name="twitter:title" content="<%= current_page.data.title %>">
+  <meta name="twitter:description" content="<%= truncate(strip_tags(current_article.body), length: 150) %>">
+  <meta name="twitter:image" content="<%= current_page.data.weekly_roundup ? image_url("assets/images/blog/unboxed_roundup.jpg") : current_page.data.main_image %>">
```

Comment:
Unfortunately we can't [test the weekly roundup image](https://cards-dev.twitter.com/validator) until it's deployed to production, but it still looks okay!
![screen shot 2016-10-26 at 15 04 53](https://cloud.githubusercontent.com/assets/885223/19729131/cc5b2d4e-9b8d-11e6-818c-bea11adb51cc.png)